ERROR - 2023-03-30 18:16:23 --> Error connecting to the database: mysqli_sql_exception: Ein Verbindungsversuch ist fehlgeschlagen, da die Gegenstelle nach einer bestimmten Zeitspanne nicht richtig reagiert hat, oder die hergestellte Verbindung war fehlerhaft, da der verbundene Host nicht reagiert hat in D:\xampp\htdocs\AGB\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 D:\xampp\htdocs\AGB\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('192.168.178.54', 'AGB', 'Agb2023!', 'agb_ausleihen_d...', 3306, '', 0)
#1 D:\xampp\htdocs\AGB\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 D:\xampp\htdocs\AGB\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#3 D:\xampp\htdocs\AGB\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 D:\xampp\htdocs\AGB\system\Model.php(223): CodeIgniter\Database\BaseBuilder->get()
#5 D:\xampp\htdocs\AGB\system\BaseModel.php(585): CodeIgniter\Model->doFindAll(0, 0)
#6 D:\xampp\htdocs\AGB\app\Libraries\LeihtHelper.php(135): CodeIgniter\BaseModel->findAll()
#7 D:\xampp\htdocs\AGB\app\Controllers\Home.php(22): App\Libraries\LeihtHelper::getUeberfaellig()
#8 D:\xampp\htdocs\AGB\system\CodeIgniter.php(927): App\Controllers\Home->index()
#9 D:\xampp\htdocs\AGB\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#10 D:\xampp\htdocs\AGB\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 D:\xampp\htdocs\AGB\index.php(55): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Ein Verbindungsversuch ist fehlgeschlagen, da die Gegenstelle nach einer bestimmten Zeitspanne nicht richtig reagiert hat, oder die hergestellte Verbindung war fehlerhaft, da der verbundene Host nicht reagiert hat in D:\xampp\htdocs\AGB\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 D:\xampp\htdocs\AGB\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 D:\xampp\htdocs\AGB\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#2 D:\xampp\htdocs\AGB\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 D:\xampp\htdocs\AGB\system\Model.php(223): CodeIgniter\Database\BaseBuilder->get()
#4 D:\xampp\htdocs\AGB\system\BaseModel.php(585): CodeIgniter\Model->doFindAll(0, 0)
#5 D:\xampp\htdocs\AGB\app\Libraries\LeihtHelper.php(135): CodeIgniter\BaseModel->findAll()
#6 D:\xampp\htdocs\AGB\app\Controllers\Home.php(22): App\Libraries\LeihtHelper::getUeberfaellig()
#7 D:\xampp\htdocs\AGB\system\CodeIgniter.php(927): App\Controllers\Home->index()
#8 D:\xampp\htdocs\AGB\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 D:\xampp\htdocs\AGB\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\xampp\htdocs\AGB\index.php(55): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-03-30 18:16:23 --> Unable to connect to the database.
Main connection [MySQLi]: Ein Verbindungsversuch ist fehlgeschlagen, da die Gegenstelle nach einer bestimmten Zeitspanne nicht richtig reagiert hat, oder die hergestellte Verbindung war fehlerhaft, da der verbundene Host nicht reagiert hat
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `leiht`
WHERE `aktiv` = :aktiv:', [...], false)
 3 SYSTEMPATH\Model.php(223): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(585): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Libraries\LeihtHelper.php(135): CodeIgniter\BaseModel->findAll()
 6 APPPATH\Controllers\Home.php(22): App\Libraries\LeihtHelper::getUeberfaellig()
 7 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Home->index()
 8 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(55): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-30 18:17:14 --> Error connecting to the database: mysqli_sql_exception: Ein Verbindungsversuch ist fehlgeschlagen, da die Gegenstelle nach einer bestimmten Zeitspanne nicht richtig reagiert hat, oder die hergestellte Verbindung war fehlerhaft, da der verbundene Host nicht reagiert hat in D:\xampp\htdocs\AGB\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 D:\xampp\htdocs\AGB\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('192.168.178.54', 'AGB', 'Agb2023!', 'agb_ausleihen_d...', 3306, '', 0)
#1 D:\xampp\htdocs\AGB\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 D:\xampp\htdocs\AGB\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#3 D:\xampp\htdocs\AGB\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 D:\xampp\htdocs\AGB\system\Model.php(223): CodeIgniter\Database\BaseBuilder->get()
#5 D:\xampp\htdocs\AGB\system\BaseModel.php(585): CodeIgniter\Model->doFindAll(0, 0)
#6 D:\xampp\htdocs\AGB\app\Libraries\LeihtHelper.php(135): CodeIgniter\BaseModel->findAll()
#7 D:\xampp\htdocs\AGB\app\Controllers\Home.php(22): App\Libraries\LeihtHelper::getUeberfaellig()
#8 D:\xampp\htdocs\AGB\system\CodeIgniter.php(927): App\Controllers\Home->index()
#9 D:\xampp\htdocs\AGB\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#10 D:\xampp\htdocs\AGB\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 D:\xampp\htdocs\AGB\index.php(55): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Ein Verbindungsversuch ist fehlgeschlagen, da die Gegenstelle nach einer bestimmten Zeitspanne nicht richtig reagiert hat, oder die hergestellte Verbindung war fehlerhaft, da der verbundene Host nicht reagiert hat in D:\xampp\htdocs\AGB\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 D:\xampp\htdocs\AGB\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 D:\xampp\htdocs\AGB\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#2 D:\xampp\htdocs\AGB\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 D:\xampp\htdocs\AGB\system\Model.php(223): CodeIgniter\Database\BaseBuilder->get()
#4 D:\xampp\htdocs\AGB\system\BaseModel.php(585): CodeIgniter\Model->doFindAll(0, 0)
#5 D:\xampp\htdocs\AGB\app\Libraries\LeihtHelper.php(135): CodeIgniter\BaseModel->findAll()
#6 D:\xampp\htdocs\AGB\app\Controllers\Home.php(22): App\Libraries\LeihtHelper::getUeberfaellig()
#7 D:\xampp\htdocs\AGB\system\CodeIgniter.php(927): App\Controllers\Home->index()
#8 D:\xampp\htdocs\AGB\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 D:\xampp\htdocs\AGB\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\xampp\htdocs\AGB\index.php(55): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-03-30 18:17:14 --> Unable to connect to the database.
Main connection [MySQLi]: Ein Verbindungsversuch ist fehlgeschlagen, da die Gegenstelle nach einer bestimmten Zeitspanne nicht richtig reagiert hat, oder die hergestellte Verbindung war fehlerhaft, da der verbundene Host nicht reagiert hat
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `leiht`
WHERE `aktiv` = :aktiv:', [...], false)
 3 SYSTEMPATH\Model.php(223): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(585): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Libraries\LeihtHelper.php(135): CodeIgniter\BaseModel->findAll()
 6 APPPATH\Controllers\Home.php(22): App\Libraries\LeihtHelper::getUeberfaellig()
 7 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Home->index()
 8 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(55): CodeIgniter\CodeIgniter->run()
